src/Tabs/TabsWrapper.php

```diff
@@ -113,9 +113,13 @@ public static function genJavaScript()
         $s = '';
         if (!defined('INCLUDED_TW_ONETIME_JS')) {
             define('INCLUDED_TW_ONETIME_JS', true);
+            $modalTitle = xla("Warning");
+            $modalContent = xla("Do you want to close the tabs");
```

Comment:
place the punctuation (`?`) within the constant

Reply:
Done! Please review it again :)